### PR TITLE
Add `date` command and update code layout

### PR DESCRIPTION
**Pull request by vdemeester**
- Add a `date` command that support getting a format (default to `RFC3339`)
- Print some errors in `os.Stderr` instead of stdout
- Relocate code command to `cmd/stupid` and update the `Makefile` to take it into account, also building a Linux binary.
- Split `main.go` in separate files

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -64,11 +75,18 @@ func printUsage() {
...
 	fmt.Println("* stupid cp SRCS DST")
+	fmt.Println("* stupid date [FORMAT]")
 	fmt.Println("* stupid rm SRCS")
 	fmt.Println("* stupid tar SRCS DST")
 	fmt.Println("* stupid untar SRC DST")
 }
 
+func date(format string) error {
+	t := time.Now()
+	fmt.Println(t.Format(format))
```
Comment (mat007) [inline main.go]: nit: could be inlined
```
fmt.Println(tine.Now().Format(format))
```

Comment (mat007) [inline main.go]: For `stupid home` we used `fmt.Print` to not include a trailing `\n`, we should probably do the same here?

File: main.go [hunk 2]
```diff
@@ -41,6 +42,16 @@ func main() {
 	case "untar":
 		checkArguments(args, 3)
 		err = untar(args[1], args[2])
+	case "date":
+		format := time.RFC3339
+		if len(args) == 1 {
+			format = args[0]
```
Comment (mat007) [inline main.go]: In my opinion we could just have RFC3339 for now and remove the format argument: «just a stupid date» 📆

Comment (vdemeester) [reply main.go]: Sounds fair 😛

File: cmd/stupid/main.go [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+			fmt.Print(home)
+		}
+	case "rm":
+		checkArguments(args, 2)
+		err = remove(args[1:])
+	case "cp":
+		checkArguments(args, 3)
+		err = copy(args[1:len(args)-1], args[len(args)-1])
+	case "date":
+		checkArguments(args, 0)
```
Comment (mat007) [inline cmd/stupid/main.go]: Not sure this actually checks anything
```
if len(args) < 0 {
```

File: README.md [hunk 4]
```diff
@@ -52,6 +53,13 @@ Example:
 ```
 stupid cp web/readme.txt web/dist/* electron/web
 ```
+
+### date
+```
+stupid date
+```
+Prints the current date with RFC3339
```
Comment (mat007) [inline README.md]: nit: needs a dot at the end

Comment (vdemeester) [reply README.md]: 😂

Comment (vdemeester) [reply README.md]: Fixed 😉

**Pull request merged**